umbraco-cloud/set-up/project-settings/public-access.md

```diff
@@ -23,11 +23,11 @@ By default, we add **Basic Authentication** to all trials and non-live environme
 1. Go to **Public Access** in the **project settings** tab
 2. Enable **Basic Authentication** on the project
    
-![Hostnames Specific Security Settings](../images/basic_auth.png)
+![Enable Basic Authentication](../images/basic_auth.png)
 
 3. Once enabled **Allow** IPs for users that need access to the project
```

Comment:
Shouldn't the highlighted text here be "Add IPs" instead of "Allow" - at least according to the image?

Reply:
Ahh good catch!

I have fixed it :)